### ISSUE TITLE
Running "npx figma connect" in a VPN environment results in a "self-signed certificate in certificate chain" error. 

### ISSUE THREAD

**Issue by ktamken** [figma/code-connect#101]
Hello.
We are trying to connect to the Figma component using Code Connect.
After executing the "npx figma connect" command and specifying the access token and working directory, when I specified the URL of the Figma component, I got the message "Failed to fetch components from Figma: self-signed certificate in certificate chain," and the connection fails.

This error occurs when executing the "npx figma connect publish" command, even if the "npx figma connect" command is successful.

Please tell me the cause of this error and how to correct it.

The environment is as follows:

OS: Windows 10 22H2
npm: 10.8.2
node_module:
    @figma/code-connect@1.0.2
    react-dom@18.3.1
    react-router-dom@6.25.0
    react@18.3.1

Network: Home network
VPN

**Comment by jimmymorris** [comment]
Also running into the same issue

**Comment by ktamken** [comment]
Hello.

@jimmymorris 
Thank you for your reply.
I understand that VPN is affecting the error in the "npx figma connect" command.
However, executing the "npx figma connect" command in a VPN-connected environment is a requirement, so I am trying to figure out what is causing the problem with the VPN and how to address it.


I will add some more information about this issue.
I checked the method defined in the link code from the error message.
https://github.com/figma/code-connect/blob/main/cli/src/connect/wizard/run_wizard.ts#L52

I understand that the process extracts the "FileKey" from the URL, creates a RESP API Endpoint, and executes the API (GET /code_connect/<FileKey>/cli_data).
I created and executed code to execute the REST API individually in the same configuration and network environment as described in the previous issue.
I expected it to end with an error as in the previous issue, but it ended normally.

Is there any difference between using the code_connect module and executing the RESP API directly?

Also, is the content of the following README relevant?
https://github.com/figma/code-connect/blob/main/README.md#common-issues
